### PR TITLE
Octavia services depend on nova and neutron

### PR DESCRIPTION
**Pull request by beagles**
The operator is needs nova and neutron to deploy the amphora controller services.

### CODE REVIEW

**Comment by beagles** [comment]
recheck

**Comment by weinimo** [comment]
recheck

File: scripts/gen-netatt.sh [hunk 1]
```diff
@@ -238,12 +238,23 @@ spec:
...
       "master": "${INTERFACE}.$((${VLAN_START}+${VLAN_STEP}*3))",
       "ipam": {
         "type": "whereabouts",
         "range": "172.23.0.0/24",
         "range_start": "172.23.0.30",
         "range_end": "172.23.0.70"
+EOF_CAT
+elif [ -n "$IPV6_ENABLED" ]; then
+    cat >> ${DEPLOY_DIR}/octavia.yaml <<EOF_CAT
+        "range": "fd00:eeee::/64",
```
Comment (weinimo) [inline scripts/gen-netatt.sh:253]: The test log shows an syntax error. https://logserver.rdoproject.org/42/742/4f1fd13a5647fb8e709451e178d0eb72706eea3c/github-check/install-yamls-crc-podified-edpm-baremetal/521adc3/job-output.txt

It looks like you need at least the line `"ipam": {` here as well

Comment (beagles) [reply scripts/gen-netatt.sh:253]: Thanks!

File: Makefile [hunk 2]
```diff
@@ -1753,12 +1753,16 @@ octavia_kuttl_run: ## runs kuttl tests for the octavia operator, assumes that ev
...
+octavia_kuttl: kuttl_common_prep ovn ovn_deploy redis_deploy neutron neutron_deploy nova nova_deploy glance glance_deploy octavia octavia_deploy_prep ## runs kuttl tests for the octavia operator. Installs octavia operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
+	make wait OPERATOR_NAME=neutron
+	make wait OPERATOR_NAME=nova
+	make wait OPERATOR_NAME=glance
 	$(eval $(call vars,$@,octavia))
 	make wait
 	make octavia_kuttl_run
 	make deploy_cleanup
 	make octavia_cleanup
+	make neutron_cleanup
```
Comment (abays) [inline Makefile:1765]: Is there any reason  `nova_cleanup` and `glance_cleanup` aren't called here as well, since you added them above?

**Review by abays — APPROVED** [review]
/lgtm

Tested locally with workarounds for missing upstream image SHAs (which are not related to the changes in this PR)

**Pull request merged**